golang/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14.2
+FROM golang:1.15
```

Comment:
wonder if it is better to have full version.

Reply:
Added patch version